### ISSUE TITLE
is there any interest in a pure es modules PR?

### ISSUE THREAD

**Issue by mreinstein** [feross/hyperscript-attribute-to-property#12]
It would be great to publish an experimental/alpha build of hyperx that is a pure es module (no commonjs.) It's still experimental in node v14 but if we were to publish a tagged package it might enable more experimentation. 

Would you be open to a PR? Happy to submit one.

**Comment by feross** [comment]
Sure, feel free to send a PR. I've never converted a package to es modules before and this is about the simplest possible package to do that on. :)

**Comment by mreinstein** [comment]
yeah, sorry for the noise of issue opening/closing. What I'm struggling with right now is the "best" way to approach this. I'm exploring how to publish code to both npm and to deno.

**Comment by mreinstein** [comment]
My struggles over the "best" way have largely been resolved by all supported versions of node now understanding the ESM dialect.  PR forthcoming.

**Comment by mreinstein** [comment]
@feross sorry for the crazy reply delay.  Let me know if you have any issues with https://github.com/feross/hyperscript-attribute-to-property/pull/13

I didn't update the `version` field in `package.json` but if this PR is accepted, we should semver major bump to 2.0.0
